portal/portal-service-impl/impl/src/sql/oracle/bullhorn_tables.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE SOCIAL_ALERTS
+(
+    ID NUMBER(19) NOT NULL AUTO_INCREMENT,
```

Comment:
`auto_increment` not supported on Oracle. You'll need to create a sequence and use a before insert trigger to pull the value from the sequence and emulate the auto_increment. Or if we are able to require Oracle 12c, use an `identity` column.

If you dont want to do that, there will need to be special handling for Oracle to insert `sequence.nextval` for the ID column.